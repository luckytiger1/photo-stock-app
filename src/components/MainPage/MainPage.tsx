import React from 'react';
import SearchBoxContainer from '../SearchBox/SearchBoxContainer';
import GalleryContainer from '../Gallery/GalleryContainer';

const MainPage = () => {
  return (
    <div>
      <SearchBoxContainer />
      <GalleryContainer />
    </div>
  );
};

export default MainPage;
