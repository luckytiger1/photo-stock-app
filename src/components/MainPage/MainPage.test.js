import React from 'react';
import { shallow } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';
import MainPage from './MainPage';

it('<MainPage /> renders correctly', () => {
  expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));
  expect(shallow(<MainPage />)).toMatchSnapshot();
});
