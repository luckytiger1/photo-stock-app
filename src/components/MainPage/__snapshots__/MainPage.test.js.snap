// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainPage /> renders correctly 1`] = `
<div>
  <Connect(SearchBoxContainer) />
  <withRouter(Connect(GalleryContainer)) />
</div>
`;
